docs/organizations/overview.mdx

```diff
@@ -5,26 +5,140 @@ description: Learn about how to create and manage Clerk organizations and their
+Organizations are a flexible and scalable way to manage users and their access to resources within your Clerk application. With organizations, you can assign specific roles and permissions to users, making them useful for managing projects, coordinating teams, or facilitating partnerships.
 
-There is no limit to the number of organizations a user can be a member of. However, a user can only create up to 100 organizations in a given application instance.
+
+<Callout type="info">
+  To explore organizations in Clerk, check out this demo repo:
+  https://github.com/clerk/organizations-demo
+</Callout>
 
 ## Enable organizations in your application
 
+Organizations are disabled by default.
+
 To enable organizations:
-1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=organizations-settings) and select your application.
+
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=organizations-settings).
 2. In the navigation sidebar, select **Organizations Settings**.
 3. Toggle on **Enable Organizations**.
 
-## Implement organization features in your application
+Once organizations are enabled, you will be presented with the default settings, roles, and permissions that are applied to all organizations in that application instance. The following sections will explain these settings in more detail.
 
-Clerk provides [prebuilt components](/docs/components/overview), [React hooks](/docs/references/react/use-organization), and [APIs](/docs/references/javascript/organization/organization) to help you integrate organizations into your application. Using the [Organizations Backend API](https://clerk.com/docs/reference/backend-api/tag/Organizations), you can provide additional [private and public metadata for the organization](/docs/organizations/metadata) and set custom attributes.
+### Roles and permissions
 
-<Callout type="info">
-  To explore Clerk's Organizations, check out this demo repo:
-  https://github.com/clerk/organizations-demo
-</Callout>
+Roles determine a user's level of access and permissions within an organization. [Learn more about how roles and permissions work and how to create your own with Clerk.](/docs/organizations/roles-permissions)
+
+### Default membership limit
+
+There is no limit to the number of organizations a user can be a member of.
+
+However, there is a limit to how many members total can be in a single organization. By default, the membership limit is set to 5 members. To change this limit, scroll to the **Default membership limit** section and update the membership limit.
+
+If you are on the Free plan, you can update the membership limit to a maximum of 5 members.
+
+If you have the Pro plan, you can set the membership limit to unlimited.
+
+You can also change this limit on a per-organization basis:
```

Comment:
```suggestion
You can also change this limit on a per-organization basis.
```
Changing to a period because the instructions are already added above with my suggestion